--- data/helper.php	2009-02-13 16:26:25.000000000 +0200
+++ data/helper.php	2009-07-28 16:50:23.128325628 +0300
@@ -105,6 +105,10 @@
                 case 'nspage':
                     $outs[] = $R->internallink(":$key:$val",NULL,NULL,true);
                     break;
+                case 'interwiki':
+					list($wikiName, $wikiUri) = explode('>', $val, 2);
+                    $outs[] = $R->interwikilink($val, NULL, $wikiName, $wikiUri, true);
+                    break;
                 case 'mail':
                     list($id,$title) = explode(' ',$val,2);
                     $id = obfuscate(hsc($id));
